pkg/cli/admin/nodeimage/create.go

```diff
@@ -500,6 +623,19 @@ func (o *CreateOptions) createRolesAndBindings(ctx context.Context) error {
 					"get",
 				},
 			},
+			{
+				APIGroups: []string{
+					"certificates.k8s.io",
+				},
+				Resources: []string{
+					"certificatesigningrequests",
+					"clusterversions",
+				},
+				Verbs: []string{
+					"get",
+					"list",
+				},
+			},
```

Comment:
@rwsu can you please also add another rule specifically for the secrets, so that they could also be created? ie

```
...
{
			APIGroups: []string{
				"",
			},
			Resources: []string{
				"secrets",
			},
			Verbs: []string{
				"get",
				"list",
				"create",
			},
		},
...
```

Otherwise [this incoming change](https://github.com/openshift/installer/pull/8717/files#diff-174c7f0a2bed95144baa2b38b752cb3a48aab7bc4380c16028faed7dd97924f9R269)  from @pawanpinjarkar  PR will break the commands

Reply:
Thanks for the heads up. We will also need to [include update](https://github.com/openshift/installer/pull/8717/files#diff-174c7f0a2bed95144baa2b38b752cb3a48aab7bc4380c16028faed7dd97924f9R284).